lib/stacks/iam.ts

```diff
@@ -719,22 +739,14 @@ export class IAMStack extends Stack {
                             `arn:${this.partition}:elasticmapreduce:${Aws.REGION}:${this.account}:cluster/*`,
                         ],
                     }),
-                    /*
-                    * Action required for auto detecting language for translate jobs
-                    */
-                    new PolicyStatement({
-                        effect: Effect.ALLOW,
-                        actions: ['comprehend:DetectDominantLanguage'],
-                        resources: ['*'],
-                    }),
                 ],
             });
 
-            if (props.enableTranslate) {
-                appPolicy.addStatements(translateIAMPermissionsPolicyStatement);
```

Comment:
This is already added in the notebook policy and was duplicative. Consolidated it to save policy space. Verified that translate still works in the UI